utils/__tests__/dashboard-helper.test.js

```diff
@@ -20,51 +22,60 @@ jest.mock('../lib/github-api-helpers', () => {
 jest.mock('node-fetch');
 
 describe('dashboard-helper', () => {
-  describe('checkLine', () => {
+  describe('getWarnings', () => {
     test('handles empty string input', () => {
-      expect(checkLine('')).toHaveLength(0);
+      expect(getWarnings('')).toHaveLength(0);
     });
 
     test('finds guid', () => {
-      expect(checkLine(`(entity.guid)`)).toHaveLength(1);
+      expect(getWarnings(`(entity.guid)`)).toHaveLength(1);
     });
 
     test('finds entityGuid', () => {
-      expect(checkLine(`entityGuid`)).toHaveLength(1);
+      expect(getWarnings({'entityGuid': 1234})).toHaveLength(1);
     });
 
     test('finds linkedEntityGuids', () => {
-      expect(checkLine(`"linkedEntityGuids": 123456`)).toHaveLength(1);
+      expect(getWarnings({"linkedEntityGuids":123456})).toHaveLength(1);
     });
 
     test('does not find null linkedEntityGuids', () => {
-      expect(checkLine(`"linkedEntityGuids": null`)).toHaveLength(0);
+      expect(getWarnings({"linkedEntityGuids":null})).toHaveLength(0);
     });
 
     test('finds permissions field', () => {
-      expect(checkLine(`"permissions": test`)).toHaveLength(1);
+      expect(getWarnings({"permissions":test})).toHaveLength(1);
     });
 
     test('finds accountId', () => {
-      expect(checkLine(`"accountId": 123456`)).toHaveLength(1);
+      expect(getWarnings({"accountId":123456})).toHaveLength(1);
     });
 
     test('does not find accountId equal to zero', () => {
-      expect(checkLine(`"accountId": 0`)).toHaveLength(0);
+      expect(getWarnings({"accountId":0})).toHaveLength(0);
     });
 
     test('finds accountIds', () => {
-      expect(checkLine(`"accountIds": [ 0 ]`)).toHaveLength(1);
+      expect(getWarnings({"accountIds":[  31415  ]})).toHaveLength(1);
     });
 
     test('does not find accountIds equal to []', () => {
-      expect(checkLine(`"accountIds": []`)).toHaveLength(0);
+      expect(getWarnings({"accountIds":[]})).toHaveLength(0);
     });
+
+    test('finds warnings across multiple lines', () => {
+      const multiLine = {
+        "accountIds": [ 
+          12345678 
+        ],
+      }
+      expect(getWarnings(multiLine)).toHaveLength(1);
```

Comment:
do we know if the `getWarnings` returns the `multiLine` object with line breaks??

Reply:
jk i'm seeing that it's a wrapper around `neatJSON` - what does `neatJSON` do?

Reply:
It's like a more powerful alternative to JSON.stringify that I'm using to collapse unnecessary line breaks so that multi-line accountIds (for example) end up as one line to check